models/event.js

```diff
@@ -60,7 +60,12 @@ const MODEL = {
         .example({
             nodes: [{ name: '~commit' }, { name: 'main' }, { name: 'publish' }],
             edges: [{ src: '~commit', dest: 'main' }, { src: 'main', dest: 'publish' }]
+        }),
+    pr: Joi
+        .object().keys({
+            url: Joi.string().description('url for PR link').example('https://git.ouroath.com/MAILSERVICES/gdpr-event-processor/pull/1')
```

Comment:
Can you put `.description` and `.example` on newlines so it's a little easier to read?